src/main/java/org/codebusters/audiogeek/spotifygazer/application/getreleases/GetReleasesRestAdapter.java

```diff
@@ -0,0 +1,73 @@
+package org.codebusters.audiogeek.spotifygazer.application.getreleases;
```

Comment:
lets divide sub-packages inside application package to indicate responsibility. Meaning:

application --> rest --> getreleases